apps/desktop-wallet/.eslintrc.js

```diff
@@ -17,5 +17,9 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 module.exports = {
-  extends: ['@alephium/eslint-config/base', '@alephium/eslint-config/react']
+  extends: [
+    '@alephium/eslint-config/base',
+    '@alephium/eslint-config/react',
+    'plugin:@tanstack/eslint-plugin-query/recommended'
```

Comment:
Adding the recommended linter to avoid some pitfalls https://tanstack.com/query/latest/docs/framework/react/installation#recommendations